templates/tags/SageFinancialProgramCodes.json

```diff
@@ -0,0 +1,156 @@
+[
```

Comment:
forgot to put "Indirects" on this list.  is there a number for this?

Reply:
No, it's not tied to a program code. It can be added here, but I wonder if it should be filtered out for the SC. It's mostly intended for infra costs that aren't tied to a particular program (like hosting the Tower web app). 

@brucehoff: What do you think?

Reply:
Good point. I'd forgotten about this. Let's add `NO PROGRAM | 00000` to the list. (I'd like to avoid the word "Indirect" which relates to how Finance uses the information as opposed to how resources are categorized.)

Reply:
done